force-app/main/default/classes/IndividualBucketAccountTests_TEST.cls

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright (c) 2021 Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @description Organization Account tests in NPSP.
+*/
+@IsTest
+private class IndividualBucketAccountTests_TEST {
+
+    @IsTest
+    static void delete_one_contact_from_bucket_account_should_not_rename_account() {
+        // Arrange: GIVEN the App is configured to use the Individual Bucket Account
+        UTIL_CustomSettingsFacade.getContactsSettingsForTests(
+                new npe01__Contacts_And_Orgs_Settings__c
+                        (npe01__Account_Processor__c = CAO_Constants.BUCKET_PROCESSOR));
+
+        // Arrange: AND the Bucket Account has multiple child Contacts
+        Contact orgContact1 = new Contact(LastName = 'testCon1');
```

Comment:
Does the bucket account automatically get created when we create the first contact? Noticed that we didn't insert an account here first

Reply:
Yep! via LegacyHouseholds#attachToBucketAccount

I'll add to an [upcoming WI](https://gus.lightning.force.com/lightning/r/a07EE00000TxulVYAR/view) that this method should begin using this new IndividualBucketAccount entity instead of doing the on-the-fly insertion of the bucket account inside this method: https://github.com/SalesforceFoundation/NPSP/blob/c105514d755197a5d3565fedf5f866ac24856040/force-app/main/domain/LegacyHouseholds.cls#L181